modin/core/io/text/custom_text_dispatcher.py

```diff
@@ -27,17 +27,17 @@ class CustomTextExperimentalDispatcher(TextFileDispatcher):
 
     @classmethod
     def _read(cls, filepath_or_buffer, columns, custom_parser, **kwargs):
-        """
+        r"""
         Read data from `filepath_or_buffer` according to the passed `read_custom_text` `kwargs` parameters.
 
         Parameters
         ----------
         filepath_or_buffer : str, path object or file-like object
             `filepath_or_buffer` parameter of `read_custom_text` function.
-        columns : list or callable(file-like object, **kwargs) -> list
+        columns : list or callable(file-like object, \*\*kwargs -> list
```

Comment:
Sphinx recognizes `**` as bold text start and warns when can't find closing `**`, we need to escape these chars.